Platform/CoffeelakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -153,11 +153,10 @@ EarlyPlatformDataCheck (
   STITCH_DATA          *StitchData;
 
   // Stitching process might pass some plafform specific data.
-  StitchData = (STITCH_DATA *)(0xFFFFFFF4);
+  StitchData = (STITCH_DATA *) 0xFFFFFFF4;
   if (StitchData->Marker != 0xAA) {
-    // No data, set default debug port to 2
     // PlatformID will be deferred to be detected
-    SetDebugPort (0xFF);
+    SetDebugPort ( PcdGet8 (PcdDebugPortNumber));
```

Comment:
The default is 0xFF.  So it needs to add it into BoardConfig.py for CFL platform.

Reply:
Done